### PR TITLE
n18772: add notes and questions

### PR DESCRIPTION
**Pull request by robot-dreams**

### CODE REVIEW

**Comment by jnewbery** [comment]
I announced the next two weeks' review clubs without any notes, so I've had to rebase this on master.

**Comment by jnewbery** [comment]
Pushed a commit that makes some minor formatting and style changes. Let me know what you think.

We don't usually publish the notes and questions until the Friday before the review club so there's no rush on this.

**Comment by robot-dreams** [comment]
Thanks for rebasing and cleaning things up. Your suggestions look good to me!

File: _posts/2020-09-30-#18772.md [hunk 1]
```diff
@@ -10,14 +10,58 @@ status: upcoming
...
+* This PR updates the `getblock` RPC to include an extra "fee" field for transactions whenever:
+
+    - `verbosity >= 2`
+    - Undo data (which can be used for efficient fee calculation) exists for that block
+
+* We discussed block undo data and how it's stored on disk in [a previous review club meeting](/17994).
+
+* Some extra changes were added to prevent fuzz test failures; we discussed
+  fuzz testing and how it's used in Bitcoin Core in [a previous review club
+  meeting](./17860).
```
Comment (jonatack) [inline _posts/2020-09-30-#18772.md:28]: untested, but `(./17860).` -> `(/17860).`?

Comment (jnewbery) [reply _posts/2020-09-30-#18772.md]: done

**Comment by jnewbery** [comment]
I've rebased this on master to pick up the ruby changes.

**Pull request merged**